README.md

```diff
@@ -1,201 +1,109 @@
-<a name="readme-top"></a>
-
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-OPTIONAL SECTIONS:
-- FAQ
-
-After you're finished please remove all the comments and instructions!
--->
-
-<div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
-
-  <h3><b>Microverse README Template</b></h3>
-
-</div>
-
 <!-- TABLE OF CONTENTS -->
 
 # 📗 Table of Contents
 
-- [📖 About the Project](#about-project)
-  - [🛠 Built With](#built-with)
-    - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
-- [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
-  - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
-- [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
-- [📝 License](#license)
+- [📗 Table of Contents](#-table-of-contents)
```

Comment:
_note: Since there is no open PR in the frontend project for me to review, I'll include my comments for that [project](https://github.com/inspecta/bridal-cars-frontend) here._

# Frontend review comments

- [ ] Good job adding frontend tests!  However, when I try to run them, this happens:

![image](https://user-images.githubusercontent.com/57421823/224254133-0fc453f6-811e-4ae9-bb3f-d7b8bd09171e.png)

Kindly correct these 3 failing tests so your app's integrity is secured. 💪 

- [ ]  Please, kindly refactor these functions to ES6 arrow functions, so your code complies with the latest standards.

These are the functions, refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

```javascript
const AlertComponent= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in VsCode for the word "function" and it will find all your non ES6 functions like this (found 2):

![image](https://user-images.githubusercontent.com/57421823/224255447-d1a88f4f-60a2-4ef3-8bd7-00653b17d2c5.png)